### PR TITLE
Fixed #34038 -- Improve link readability and identifiability

### PR DESCRIPTION
**Pull request by CLWilliam**
To improve the readability of links in the Django admin:
- The default colour has been slightly darkened in light mode (-1% lightness in HSL colour space)
- The links are underlined on hover for all default links (add/create links, breadcrumbs...) to be more identifiable for people with colour vision deficiencies.
- Links in the form rows have been underlined by default as - since they bend into the help text - they are very difficult to distinguish from plain text.

### CODE REVIEW

**Review by knyghty — COMMENTED** [review]
Sorry for the delay. This looks good, but I think we can remove the hover style, I think it looks better without and doesn't provide much value IMO.

**Comment by felixxm** [comment]
@CLWilliam Do you have time to keep working on this? (see https://github.com/django/django/pull/16101#pullrequestreview-1135339000)

**Comment by CLWilliam** [comment]
@felixxm it's on the menu for this evening!

File: django/contrib/admin/static/admin/css/base.css [hunk 1]
```diff
@@ -24,7 +24,7 @@ html[data-theme="light"],
     --breadcrumbs-link-fg: var(--body-bg);
     --breadcrumbs-bg: var(--primary);
 
-    --link-fg: #447e9b;
+    --link-fg: #437c98;
```
Comment (felixxm) [inline django/contrib/admin/static/admin/css/base.css:27]: The new color have enough contrast when the background is white. Unfortunately `--link-fg` is also used with yellow (`#ffffcc`) and gray backgrounds (`#f8f8f8`). We should make it even darker to meet AA standards, e.g. `#417893` (\cc @knyghty.)

**Comment by felixxm** [comment]
Closing due to inactivity.

**Pull request closed without merging**